x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -24943,13 +24943,11 @@
     "xpack.integrationAssistant.step.dataStream.integrationNameDescription": "このパッケージ名は、Elasticのインジェストパイプラインの統合を参照するために使用されます",
     "xpack.integrationAssistant.step.dataStream.integrationNameTitle": "パッケージ名を定義",
     "xpack.integrationAssistant.step.dataStream.logsSample.description": "ファイルをドラッグアンドドロップするか、ファイルを参照します",
-    "xpack.integrationAssistant.step.dataStream.logsSample.description2": "JSON/NDJSON形式",
```

Comment:
It looks like this warning is still present – why is the translation gone?

Reply:
The `i18n_check` script run was missed. Ran it now.